ballerina/tests/resources/expected_results/block_strings_with_invalid_characters.json

```diff
@@ -0,0 +1,13 @@
+{
+    "errors": [
+        {
+            "message": "Syntax Error: Cannot parse the unexpected character \"\"\".",
```

Comment:
Why does this consider an invalid character?

Reply:
This error is not coming from the block string reading. Since this input block string includes a ```""""``` in the middle, rest of the string consider as an identifier. Then this first ```"``` at the end of the the string consider as an invalid character since identifier can't have ```"``` as a character.

Reply:
@ThisaruGuruge is this the correct scenario?

Reply:
I still don't understand why is this considered an invalid character (or character sequence).

IMHO, any character sequence is accepted within the block string, except the `"""`, which terminates the block.

Even if you try the same block string in the Appollo server, it does not recognize this as an invalid character. What's the basis of considering this an invalid character?